### PR TITLE
Matt&Billy/APPEALS-10172

### PR DESCRIPTION
**Pull request by ThorntonMatthew**
Resolves [APPEALS-10172](https://vajira.max.gov/browse/APPEALS-10172)

### Description
Billy noticed a bug where, whenever cancelling an intake while on the "Add Claimant" page, it would not redirect you back to the Intake homepage. This could cause users to seemingly become trapped (though, workarounds were confirmed to exist). Continuing to proceed with form population after the failed redirect could cause further errors, but no noticeable harm.

Edit: This issue has been observed on the AddPoaPage as well.

### Acceptance Criteria
- [ ] Users are redirected from the "Add Claimant" page to the form selection page of intake whenever they cancel an intake while on the "Add Claimant" form.
- [ ] Users are redirected from the "Add Claimant's POA" page to the form selection page of intake whenever they cancel an intake while on the "Add Claimant's POA" form.

### Testing Plan
1. Log into Caseflow as an Intake user (ex: `BVADWISE`)
2. Navigate to the Intake homepage ([Local link](http://localhost:3000/intake/))
3. Begin a 10182 intake.
4. On the Review page, select "Yes" for the prompt "Is the claimant someone other than the Veteran?".
5. Select "Claimant not listed" in the radio fieldset that will appear as a result.
localhost_3000_intake_add_claimant (6)

6. Once the required fields are populated, click the "Continue to next step" button.
7. You will now be on the "Add Claimant" page.
localhost_3000_intake_add_claimant (7)

8. Click on "Cancel intake". Select any of the cancellation reasons, and then click the submission button.
localhost_3000_intake_add_claimant (8)

9. Ensure you are redirected back to the form selection page (`/intake/`).

10. Repeat steps 1-7.
11. This time, select a relationship from the "Relationship to the Veteran" dropdown.
12. Fill in all required fields. Select "Yes" for the prompt "Do you have a VA Form 21-22 for this cl
aimant?"
localhost_3000_intake_add_claimant (9)

13. You will now be on the "Add Claimant's POA" page (`AddPoaPage.jsx`)
localhost_3000_intake_add_claimant (11)

14. Click on "Cancel intake". Like before, select any of the options on the modal and then click on the submission button.
15. Ensure you are redirected back to the form selection page (`/intake/`).

- [ ] For higher-risk changes: [Deploy the custom branch to UAT to test](https://github.com/department-of-veterans-affairs/appeals-deployment/wiki/Applications---Deploy-Custom-Branch-to-UAT)

### CODE REVIEW

File: Makefile.example [hunk 1]
```diff
@@ -69,9 +69,6 @@ down:	## Stop the docker containers
 down-m1:  ## Stop the docker containers on a M1 Mac
 	docker-compose -f docker-compose-m1.yml down
 
-down-m1:  ## Stop the docker containers on a M1 Mac
-	docker-compose -f docker-compose-m1.yml down
-
```
Comment (ajspotts) [inline Makefile.example:74]: @ThorntonMatthew this is unrelated and simply a removal of a duplicate `make` command correct?

Comment (ThorntonMatthew) [reply Makefile.example:74]: That's correct!

**Review by ajspotts — APPROVED** [review]
lgtm!

**Comment by ajspotts** [comment]
Closing in favor of https://github.com/department-of-veterans-affairs/caseflow/pull/17925
